lib/webpage_cache_service.rb

```diff
@@ -0,0 +1,133 @@
+class WebpageCacheService
+  DEFAULT_HEADERS = {
+    user_agent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:57.0) Gecko/20100101 Firefox/57.0",
```

Comment:
cheeky

Reply:
Yeah I suppose I could make my own user agent and be more informative. But then do you also handle robots.txt and such too?